install.markdown

```diff
@@ -3,37 +3,29 @@ layout: page
 title: Install IntelliJ Rust
 ---
 
-## Getting alpha builds
+## Alpha builds
 
-IntelliJ Rust is currently in the _alpha_ stage of development. When we reach some acceptable level of stability,
-we publish alpha builds to the IntelliJ's main plugin [repository](https://plugins.jetbrains.com/plugin/8182).
-You can download and install these builds directly from `Plugins > Browse repositories` screen in your IDE.
+IntelliJ Rust is in _alpha_ stage. We regularly publish stable builds to the JetBrains plugin [repository](https://plugins.jetbrains.com/plugin/8182).
+You can download and install these builds directly from `Settings / Preferences | Plugins | Marketplace` in your IDE.
 
-As this is a preview release, something might go wrong. So, be brave to face some nasty bugs.
-If you find one, we would very appreciate if you file an [issue](https://github.com/intellij-rust/intellij-rust/issues)
-on our bugtracker or ping us on our [chat](https://gitter.im/intellij-rust/intellij-rust).
+For bug reports and feature requests, please use our [issue tracker](https://github.com/intellij-rust/intellij-rust/issues).
 
-## Setting up nightly builds
+## Nightly builds
 
-In addition to preview releases, we also
-ship pre-release builds of _intellij-rust_ and _intellij-toml_ in the nightly channel.
+In addition to aplha releases, we also ship pre-builds of _intellij-rust_ and _intellij-toml_ in the nightly channel.
 
-To use them you need to follow the
-[instructions](https://www.jetbrains.com/idea/help/managing-enterprise-plugin-repositories.html)
-of adding additional plugin repository and paste the URL for the one you need:
+To use them, follow this
+[instruction](https://www.jetbrains.com/idea/help/managing-enterprise-plugin-repositories.html#repos)
+for adding a custom plugin repository, and use one of the following URLs:
 
 ```yaml
 - Rust: https://plugins.jetbrains.com/plugins/nightly/8182
 - TOML: https://plugins.jetbrains.com/plugins/nightly/8195
 ```
 
-Nightly builds are daily snapshots of current development branch. So, it's natural that there may be even
-more bugs than in alpha channel. If you find one, we would very appreciate if you file an
-[issue](https://github.com/intellij-rust/intellij-rust/issues) on our bugtracker or ping us on our
-[chat](https://gitter.im/intellij-rust/intellij-rust).
+Nightly builds are daily snapshots of the current development branch, so there may be more bugs than in the alpha channel. If you find one, please file an
+[issue](https://github.com/intellij-rust/intellij-rust/issues) in our tracker.
```

Comment:
What do you think about changing link to issue list with `https://github.com/intellij-rust/intellij-rust/issues/new`?
Also, maybe it makes sense to add some note about our `Create New Issue` action?